### PR TITLE
Minor edits, 12/31/2015

### PR DESCRIPTION
**Pull request by Mark9013100**
Adds the white cap to white wardrobes.
Adds black scrubs to the medical wardrobe.
Comments out the santa hats in custom loadout.
Minor change to the long forensics jacket names in custom loadout.
Removes the punctuation from the chef's classic apron.
Minor Z-1 area correction.
Re-adds maintenance access back to Medical Doctors, due to the map's size, and considering every other emergency role has it. They also had it prior to the introduction of the paramedic role.

### CODE REVIEW

File: code/modules/client/preferences_gear.dm [hunk 1]
```diff
@@ -233,6 +233,7 @@ var/global/list/gear_datums = list()
 	cost = 1
 	slot = slot_head
 
+/*
 /datum/gear/santahat
```
Comment (Zuhayr) [inline code/modules/client/preferences_gear.dm]: This may cause issues with runtimes upon trying to spawn if people have the item saved.

**Comment by Mark9013100** [comment]
Right, noted. Would flat out deleting it work better?

**Comment by Zuhayr** [comment]
No. It might be best to just increase the point cost and leave the item in.

**Comment by Hubblenaut** [comment]
Most expensive santa hat in the world

**Comment by Zuhayr** [comment]
Conflicts.

**Comment by Mark9013100** [comment]
Considering I've no idea what's causing the conflict, I'll just close this PR and open another.

**Comment by Hubblenaut** [comment]
It's the map file, just remerge it

**Pull request closed without merging**